app/src/main/java/com/github/onlynotesswent/model/users/UserRepositoryFirestore.kt

```diff
@@ -96,11 +97,32 @@ class UserRepositoryFirestore(private val db: FirebaseFirestore) : UserRepositor
   }
 
   override fun deleteUserById(id: String, onSuccess: () -> Unit, onFailure: (Exception) -> Unit) {
-    db.collection(collectionPath)
-        .document(id)
-        .delete()
-        .addOnSuccessListener { onSuccess() }
-        .addOnFailureListener { exception -> onFailure(exception) }
+    getUserById(
+        id,
+        { user ->
+          // for each of the current user follower remove them from their following list
+          user.friends.followers.forEach { follower ->
+            Log.e("RemovingMyselfFromFollower", follower)
```

Comment:
I don't think you need to log each time a follower is removed, since this would create spam in the logs

Reply:
Yep it was leftover debuging code, thanks for the catch !